charts/kubearchive/templates/database/ps-secret.yaml

```diff
@@ -5,9 +5,9 @@ kind: Secret
 type: Opaque
 metadata:
   labels:
-    app: {{ .Values.database.name | default "postgresql" }}
-  name: {{ .Values.database.name | default "postgresql"}}-secret
-  namespace: {{ .Values.kubearchive.namespace }}
+    app: {{ tpl .Values.database.name . }}
+  name: {{ tpl .Values.database.name . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I suggest removing this based on the fact that the `kubearchive` namespace is provided by the `-n` option in `helm install` command.

```suggestion
```

Reply:
removed the `namespace`